content/posts/2025-vision.md

```diff
@@ -0,0 +1,97 @@
+title: Read the Docs 2025 Vision: great support for all documentation tools
+date: 2025-01-01
+description: Our vision for Read the Docs in 2025, supporting all documentation tools and enhancing frontend features with Addons.
+category: Meta
+tags: addons, vision, builders
+authors: Eric Holscher
+status: published
+image: /images/2025-vision.jpg
+image_credit: Photo by <a href="https://unsplash.com/@mcnoble?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Matt Noble</a> on <a href="https://unsplash.com/photos/selective-focus-photo-of-silver-tower-viewer-telescope-facing-sunshine-BpTMNN9JSmQ?utm_content=creditCopyText&utm_medium=referral&utm_source=unsplash">Unsplash</a>
+
+As we look forward to 2025,
+we took some time to reflect on where we've been and where we're going,
+and wanted to share that with you.
+
+**Our vision for Read the Docs is to be the best place to host software documentation using a docs-as-code approach**.
+This means that we're building on top of Git,
+and providing features that make sense for software documentation.
+
+**Our product philosophy is to provide a great default experience but allow users to customize and extend as needed.**
+This means that we provide a set of features that work for most projects,
+but then allow you to extend or replace them as needed.
+
+Below is a breakdown of how that plays out with each of the major parts of Read the Docs.
+
+## Major steps in 2024
+
+We've been working to expand build support beyond Sphinx and MkDocs,
+and build Addons for powerful frontend features that work on any tool.
+
+The major steps here have been:
+
+* **Fully redesigned dashboard:** We've been working on a [new dashboard](https://about.readthedocs.com/blog/2024/11/rollout-of-our-new-dashboard/) that brings our site into the modern era. This has been fully rolled out to all users as the default, and we're continuing to improve it.
+* **Removing build magic:** Removing the injection of data or configuration into your documentation builds. We used to have a full Sphinx extension that we injected, and a lot of configuration file manipulation for both [Sphinx](https://about.readthedocs.com/blog/2024/07/addons-by-default/) and [MkDocs](https://about.readthedocs.com/blog/2024/03/mkdocs-yaml-manipulation/). This has been removed, which means builds should happen exactly as they do locally.
+* **Adding build support for all tools:** We built support into our build system for [fully custom builds](https://blog.readthedocs.com/build-customization/), which allows any build process that outputs HTML to be hosted on Read the Docs.
+* **Adding frontend support for all tools:** We rebuilt our Flyout and JavaScript that is injected into your documentation builds as a set of [Read the Docs Addons](https://about.readthedocs.com/blog/2024/04/enable-beta-addons/). This enables granular configuration.
```

Comment:
I think some folks may not know what a "flyout" is. I'd define it even if just in a few words. Or you could say something like "more details on that below".